packages/manager/apps/pci-public-ip/src/pages/onboarding/OnBoardingPage.tsx

```diff
@@ -0,0 +1,200 @@
+import {
+  useEnvironment,
+  useNavigation,
+} from '@ovh-ux/manager-react-shell-client';
+import {
+  Card,
+  OnboardingLayout,
+  PciDiscoveryBanner,
+  isDiscoveryProject,
+} from '@ovhcloud/manager-components';
+import { OsdsBreadcrumb, OsdsText } from '@ovhcloud/ods-components/react';
+import { useEffect, useState } from 'react';
```

Comment:
I have the same imports organisation rules, i don't understand why i was blocked on my previous PR !! is there a rules somewhere that we're not aware of ?